spec/features/visitors/sign_up_spec.rb

```diff
@@ -255,19 +242,6 @@
     expect(current_path).to eq user_confirmation_path
   end
 
-  context 'confirmation instructions sent to existing user', email: true do
-    xit 'does not send an email to the existing user' do
```

Comment:
excellent, thanks @jessieay -- glad we asked.

github issues are good for tracking things over time. pending tests are good ways to annoy us into having the conversation. :)